src/Tests/Entities/ProjectTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace ProjectManagementApi\Tests\Entities;
+
+use DateInterval;
+use DateTime;
+use ProjectManagementApi\Entities\Project;
+use PHPUnit\Framework\TestCase;
+require '../../../vendor/autoload.php';
+
+class ProjectTest extends TestCase
+{
+    public function testSuccessConstructor_overdueTrue()
+    {
+        //creating a deadline that is 1 year before the current date
+        $deadline = new DateTime();
+        $dateInterval = new DateInterval('P1Y');
+        $dateInterval->invert = 1;
+        $deadline->add($dateInterval);
+        
+        $testProject = new Project(1, 'name', 1, $deadline);
+        $actualOutput = $testProject->getIsOverdue();
+        $expectedOutput = true;
```

Comment:
in this case "output" is actually not a semantic word to use, because this is not the output of the constructor (the constructor doesn't output anything). a better name might be `$expectedIsOverdue` & `$actualIsOverdue`